app/Modules/Person/Models/Person.php

```diff
@@ -116,15 +116,15 @@ class Person extends Model implements HasLogEntries
         'gender_id' => 'integer',
         'birth_year' => 'integer',
         'ethnicities' => 'array',
-        'ethnicity_opt_out' => 'boolean',
+      //  'ethnicity_opt_out' => 'boolean',
```

Comment:
If you mark these as boolean, can't you avoid casting on the javascript side?

Reply:
No.  I coded the solution for and reimplemented.  Casting doesn't work with Boolean.

Reply:
I'm not entirely sure what issue you were running into when you say that casting doesn't work with boolean-- it is listed as a supported casting type in laravel (https://laravel.com/docs/10.x/eloquent-mutators). I suspect that there may have been some *_opt_out fields that were left out of the casting block when you had done your prior testing. We can revisit this after other things are resolved, though.